sql/planarGraph/boyerMyrvold.sql

```diff
@@ -35,7 +35,9 @@ CREATE OR REPLACE FUNCTION pgr_boyerMyrvold(
     OUT seq BIGINT,
     OUT source BIGINT,
     OUT target BIGINT,
-    OUT cost FLOAT)
+    OUT cost FLOAT,
+    OUT planarity BOOLEAN
```

Comment:
Based on:
https://www.postgresql.org/docs/12/xfunc-c.html
Remove the other all the OUT parameters